### PR TITLE
boards: nrf9160 DK: spell the board name consistently

### PR DESCRIPTION
**Pull request by mbolivar-nordic**
Use "nRF9160 DK" throughout. Change the title to "Development Kit" and
introduce the abbreviation there.

### CODE REVIEW

**Comment by mbolivar-nordic** [comment]
@ioannisg what do you think about this? I thought it would be nice to be consistent when reviewing the Innblue board: https://github.com/zephyrproject-rtos/zephyr/pull/23451#discussion_r401996777

**Review by ioannisg — APPROVED** [review]
looks ok, thanks

File: boards/arm/nrf9160dk_nrf9160/doc/index.rst [hunk 1]
```diff
@@ -1,16 +1,15 @@
 .. _nrf9160dk_nrf9160:
 
-nRF9160 DK
-##########
+nRF9160 Development Kit (DK)
```
Comment (anangl) [inline boards/arm/nrf9160dk_nrf9160/doc/index.rst:3]: Should we do same changes in documentation of nRF52840 DK and nRF51 DK?

**Review by anangl — APPROVED** [review]

File: boards/arm/nrf9160dk_nrf9160/doc/index.rst (hunk 1 above)
Comment (ioannisg) [reply boards/arm/nrf9160dk_nrf9160/doc/index.rst:3]: We should, yes. Maybe as part of renaming the remaining nrf52 boards?

**Pull request merged**